app/src/test/java/com/github/sdp/mediato/model/CollectionTests.java

```diff
@@ -0,0 +1,65 @@
+package com.github.sdp.mediato.model;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertThrows;
+
+import com.github.sdp.mediato.model.media.Collection;
+import com.github.sdp.mediato.model.media.CollectionType;
+import com.github.sdp.mediato.model.media.Movie;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CollectionTests {
+    private static Map<String, Review> reviews = new HashMap<>();
+    private static String SAMPLE_TITLE = "Harry Potter";
+    private static String SAMPLE_DESCRIPTION = "Description";
+    private static String SAMPLE_URL = "Url";
+    private static int SAMPLE_ID = 1;
+
+    private static String SAMPLE_USERNAME = "testUser";
+    private static final Movie SAMPLE_MOVIE = new Movie(SAMPLE_TITLE , SAMPLE_DESCRIPTION, SAMPLE_URL, SAMPLE_ID);

```

Comment:
Since we remove the abstract to the Media class, it might a good thing to test the constructor with two URLs for images.